src/utils/EstablishmentUtils.ts

```
@@ -0,0 +1,59 @@
+  const formattedIntervals = todayHours.intervals
+    .map((interval) => `${interval.start} às ${interval.end}`)
+    .join(', ');
+
+  return `${todayLabel}: ${formattedIntervals}`;
+}
+
+export function createAlias(name: string): string {
+  return name
+    .toLowerCase()
+    .normalize('NFD')
+    .replace(/[\u0300-\u036f]/g, '')
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)